uk.co.compucorp.civicrm.hrcore/xml/Menu/hrcore.xml

```diff
@@ -36,4 +36,10 @@
     <title>Create User Account</title>
     <access_arguments>create users</access_arguments>
   </item>
+  <item>
```

Comment:
@omarabuhussein this page will indeed be called via ajax in the CiviHR admin, as the user submenu will be injected via JS using the hrui extension, so that we don't have to touch the core files